src/components/Widgets/DateControl.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import DateTime from 'react-datetime';
+
+export default class DateTimeControl extends React.Component {
+  handleChange = (datetime) => {
+    this.props.onChange(datetime);
+  };
+
+  render() {
+    return (<DateTime
+      timeFormat={false}
+      value={this.props.value || new Date()}
+      onChange={this.handleChange}
+    />);
+  }
+}
+
+DateTimeControl.propTypes = {
+  onChange: PropTypes.func.isRequired,
+  value: PropTypes.object,
```

Comment:
PropTypes.object is not allowed in our linting rules, but i'm not sure there's any better choice for the DateControl or the DateTimeControl.

The problem here is that depending on the `formatter` the value for a date control can either be a string, a Date or a moment.js object and the control (and preview) component should do the right thing with either of those.